cypress/integration/requests/assign-tags-to-request.spec.js

```diff
@@ -0,0 +1,74 @@
+import uuid from 'uuid';
+import testType from '../../support/dictionary/testTypes';
+import TopMenu from '../../support/fragments/topMenu';
+import Requests from '../../support/fragments/requests/requests';
+import NewRequest from '../../support/fragments/requests/newRequest';
+
+
+describe('Assign Tags to Request', () => {
+  const barcode = uuid();
+  const lastName = `Test-${uuid()}`;
+
+  before(() => {
```

Comment:
Are you sure that login should be located into before? Issue can be presented in case of several test scenarios in one spec.js file